lib/endpoints/class-wp-rest-terms-controller.php

```diff
@@ -528,6 +505,63 @@ public function delete_item( $request ) {
 	}
 
 	/**
+	 * Prepare a single term for create or update
+	 *
+	 * @param WP_REST_Request $request Request object.
+	 * @return object $prepared_term Term object.
+	 */
+	public function prepare_item_for_database( $request ) {
+		$prepared_term = new stdClass;
+
+		if ( isset( $request['name'] ) ) {
+			$prepared_term->name = $request['name'];
+		}
+
+		if ( isset( $request['slug'] ) ) {
+			$prepared_term->slug = $request['slug'];
+		}
+
+		if ( isset( $request['term_group'] ) ) {
+			$prepared_term->term_group = (int) $request['term_group'];
+		}
+
+		if ( isset( $request['term_taxonomy_id'] ) ) {
```

Comment:
We don't support setting `term_taxonomy_id` through the REST API. `term_taxonomy_id` and `term_id` are the same in our eyes, and `term_id` is always exposed as `id`.